packages/react-core/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "dependencies": {
-    "@patternfly/react-styles": "^0.0.1"
+    "@patternfly/react-styles": "^1.0.0"
```

Comment:
we should not have to do this anymore until another major version.  This is only due to the issue with this not publishing correctly in the first place.